app/Makefile

```
@@ -1,4 +1,7 @@
-.PHONY: build start stop logs deploy pull setup test test-web test-e2e test-coverage test-report local local-build docs docs-python docs-ui docs-check
+.PHONY: build start stop logs deploy pull setup test test-web test-web-local venv-web test-e2e test-coverage test-report local local-build docs docs-python docs-ui docs-check
```

checkmake (0.2.2)

[warning] Missing required phony target "all"

(minphony)